README.md

```diff
@@ -17,10 +17,10 @@
     </sup>
   </div>
 
-[📘Documentation](https://mmaction2.readthedocs.io/en/1.x/) |
-[🛠️Installation](https://mmaction2.readthedocs.io/en/1.x/get_started.html) |
+[📘Documentation](https://mmaction2.readthedocs.io/en/dev-1.x/index.html) |
```

Comment:
The 1.x branch is a released branch while dev-1.x branch is a developing branch that may change every day. So I think it should be https://mmaction2.readthedocs.io/en/1.x/

Reply:
Thank you very much for your reply. Now the readthedocs page has been opened. The problems I raised have been solved